.github/workflows/cluster-faces-test.yml

```diff
@@ -136,7 +136,7 @@ jobs:
         run: |
           rm -rf apps/recognize/models
 
-      - uses: actions/cache/restore@v3
+      - uses: actions/cache/restore@actions/cache/save@5a3ec84eff668545956fd18022155c47e93e2684 # v4.2.3@v3
```

Comment:
`@v3` is extra at the end
same for other `actions/cache/restore` blocks

Reply:
looks like the new changes were not pushed :wink:

Reply:
sorry I had edited my first comment
`actions/cache/restore@actions/cache/save@5a3ec84eff668545956fd18022155c47e93e2684 # v4.2.3`
here `actions/cache/save@` should not be there

Reply:
Ah, I get it now. Fixed.